packages/tenant-process/test/tenant.integration.test.ts

```diff
@@ -1671,5 +1679,162 @@ describe("Integration tests", () => {
+            {
+              id: tenant2.id,
+              name: tenant2.name,
+              attributeId: tenantCertifiedAttribute2.id,
+              attributeName: readModelCertifiedAttribute1.name,
+            },
+          ],
+          totalCount: 2,
+        };
+
+        expect(result).toEqual(expectedResult);
+      });
+      it("should not return the attributes when they are revoked", async () => {
+        const tenantCertifiedAttribute: TenantAttribute = {
+          ...getMockCertifiedTenantAttribute(),
+          revocationTimestamp: undefined,
+        };
+        const revokedAttribute: TenantAttribute = {
+          ...getMockCertifiedTenantAttribute(),
+          revocationTimestamp: new Date(),
+        };
+
+        const readModelCertifiedAttribute: AttributeReadmodel = {
+          ...getMockReadModelAttribute(),
+          id: tenantCertifiedAttribute.id,
+          origin: tenant1.id,
+        };
+        const readModelRevokedCertifiedAttribute: AttributeReadmodel = {
+          ...getMockReadModelAttribute(),
+          id: revokedAttribute.id,
+          origin: tenant1.id,
+        };
+
+        await writeInReadmodel(readModelCertifiedAttribute, attributes);
+        await writeInReadmodel(readModelRevokedCertifiedAttribute, attributes);
+
+        const tenantWithCertifiedAttributes: Tenant = {
+          ...tenant2,
+          attributes: [
+            ...tenant2.attributes,
+            tenantCertifiedAttribute,
+            revokedAttribute,
+          ],
+        };
+
+        await addOneTenant(tenantWithCertifiedAttributes, postgresDB, tenants);
+
+        const result = await readModelService.getCertifiedAttributes({
+          certifierId: tenant1.id,
+          offset: 0,
+          limit: 50,
+        });
+
+        expect(result.results).not.toContainEqual({
+          id: tenant2.id,
+          name: tenant2.name,
+          attributeId: revokedAttribute.id,
+          attributeName: readModelRevokedCertifiedAttribute.name,
+        });
+      });
+      it("should correctly manage pagination params (offset, limit)", async () => {
+        const tenantCertifiedAttribute1: TenantAttribute = {
+          ...getMockCertifiedTenantAttribute(),
+          revocationTimestamp: undefined,
+        };
+        const tenantCertifiedAttribute2: TenantAttribute = {
+          ...getMockCertifiedTenantAttribute(),
+          revocationTimestamp: undefined,
+        };
+
+        const readModelCertifiedAttribute1: AttributeReadmodel = {
+          ...getMockReadModelAttribute(),
+          id: tenantCertifiedAttribute1.id,
+          origin: tenant1.id,
+        };
+        const readModelCertifiedAttribute2: AttributeReadmodel = {
+          ...getMockReadModelAttribute(),
+          id: tenantCertifiedAttribute2.id,
+          origin: tenant1.id,
+        };
+
+        await writeInReadmodel(readModelCertifiedAttribute1, attributes);
+        await writeInReadmodel(readModelCertifiedAttribute2, attributes);
```

Comment:
You could just use an `attribute: Attribute` and then use
```
writeInReadModel(toReadModelAttribute(attribute))
```

In this way, when we will remove the `toReadModel` adapters, it will be really easy to adapt all the involved parts.